cypress/integration/ui_spec.js

```diff
@@ -0,0 +1,685 @@
+				};
+
+				cy.intercept("POST", "*/**/devices", (req) => {
+					expect(JSON.parse(req.body))
+						.to.have.deep.property("userAdditionalInformation", newUserAdditionalInformation);
+				}).as("createDevice");
+
+				cy.window().then((win) => {
+					// eslint-disable-next-line no-param-reassign
+					win.parleySettings.userAdditionalInformation["some-layer"] = newUserAdditionalInformation["some-layer"];
+				});
+
+				cy.wait("@createDevice");
+			});
+		});
+		describe("weekdays", () => {
+			describe("format [day, start, end]", () => {
+				it("should show we are offline/online outside/inside working hours", () => {
+					const parleyConfig = {
+						weekdays: [ // closed every day
+							["Monday"],
+							["Tuesday"],
+							["Wednesday"],
+							["Thursday"],
+							["Friday"],
+							["Saturday"],
+							["Sunday"],
+						],
+						interface: {hideChatAfterBusinessHours: true},
+					};
+
+					cy.visit("/", {
+						onBeforeLoad: (win) => {
+							// eslint-disable-next-line no-param-reassign
+							win.parleySettings = parleyConfig;
+						},
+					});
+
+					cy.get("[id=app]").as("app");
+
+					// Launcher is not rendered because we are offline
+					// and outside working hours
+					cy.get("@app")
+						.get("[class^=launcher__]")
+						.should("not.exist");
+
+					// Test if it changes during runtime
+					const newWeekdays = [
+						[
+							"Monday", 8.00, 23.30,
+						],
+						[
+							"Tuesday", 8.00, 23.30,
+						],
+						[
+							"Wednesday", 8.00, 23.30,
+						],
+						[
+							"Thursday", 8.00, 23.30,
+						],
+						[
+							"Friday", 8.00, 23.30,
+						],
+						[
+							"Saturday", 8.00, 23.30,
+						],
+						[
+							"Sunday", 8.00, 23.30,
+						],
+					];
+
+					cy.window().then((win) => {
+						// eslint-disable-next-line no-param-reassign
+						win.parleySettings.weekdays = newWeekdays;
+					});
+
+					// Launcher should appear again because we
+					// are inside working hours
+					clickOnLauncher();
+				});
+			});
+			describe("format [day, start, end, bool]", () => {
+				it("should show we are offline/online outside/inside working hours", () => {
+					const parleyConfig = {
+						weekdays: [ // closed every day
+							["Monday"],
+							["Tuesday"],
+							["Wednesday"],
+							["Thursday"],
+							["Friday"],
+							["Saturday"],
+							["Sunday"],
+						],
+						interface: {hideChatAfterBusinessHours: true},
+					};
+
+					cy.visit("/", {
+						onBeforeLoad: (win) => {
+							// eslint-disable-next-line no-param-reassign
+							win.parleySettings = parleyConfig;
+						},
+					});
+
+					cy.get("[id=app]").as("app");
+
+					// Launcher is not rendered because we are offline
+					// and outside working hours
+					cy.get("@app")
+						.get("[class^=launcher__]")
+						.should("not.exist");
+
+					// Test if it changes during runtime
+					const newWeekdays = [
+						[
+							"Monday", 8.00, 23.30, true,
```

Comment:
Ik zou wel verwachten dat je ook een case met `false` in deze test zou hebben zitten. Anders test je toch niet echt veel meer dan met je vorige test?

Reply:
Done